src/electrai/dataloader/mp.py

```diff
@@ -64,21 +64,23 @@ def __init__(
         rho_type: str,
         data_augmentation: bool = True,
         random_state: int = 42,
+        tile_size: int | None = None,
     ):
         """
         Parameters
         ----------
         data: list of voxel data of length batch_size.
         rho_type: chgcar or elfcar.
-        data_size: target size of data.
-        label_size: target size of label.
-        pyrho_uf: pyrho upsampling factor
```

Comment:
These were previously removed from the function, simply updating the docstring to current parameters.